packages/cli/src/packages-to-be/validators.ts

```diff
@@ -183,15 +184,17 @@ export const getValidator = async (
     args: [address as StrongAddress],
     blockNumber: typeof blockNumber === 'undefined' ? undefined : BigInt(blockNumber),
   })
+  const scoreManager = await getScoreManagerContract({ public: client })
```

Comment:
nit: could we parallelize the validatorTuple read, tje score read and the name read ?